eng/pipelines/weekly_automation.yml

```diff
@@ -0,0 +1,30 @@
+trigger: none
```

Comment:
Lets be sure to add this yml file directly to the CODEOWNER like https://github.com/Azure/azure-sdk-for-js/blob/main/.github/CODEOWNERS#L639 so that we have someone identified to get pipeline failure notifications.